lib/futbol_data.rb

```diff
@@ -59,4 +59,51 @@ def case_is_game_teams
     @game_teams
   end
 
+  def team_info(passed_id)
```

Comment:
I like that we moved all of these into futbol since they are more like CSV behaviors instead of statistics behaviors